app/controllers/ApiController.scala

```diff
@@ -0,0 +1,100 @@
+package controllers
+
+import actions.LoginAction
+import Operators.UserOperators
+import helper.StringHelper
+import java.util.UUID
+import javax.inject.{Inject, Singleton}
+import models.EventType.DeploymentDashboardUnauthorized
+import models.{Area, User, UserGroup}
+import play.api.libs.json.Json
+import play.api.mvc.InjectedController
+import scala.concurrent.ExecutionContext
+import serializers.ApiModel._
+import services.{EventService, OrganisationService, UserGroupService, UserService}
+
+@Singleton
+case class ApiController @Inject() (
+    loginAction: LoginAction,
+    eventService: EventService,
+    organisationService: OrganisationService,
+    userService: UserService,
+    userGroupService: UserGroupService
+)(implicit val ec: ExecutionContext)
+    extends InjectedController
+    with UserOperators {
+  import OrganisationService.FranceServiceInstance
+
+  private def matchFranceServiceInstance(
+      fsInstance: FranceServiceInstance,
```

Comment:
```suggestion
      franceServiceInstance: FranceServiceInstance,
```
je ne suis pas trop fan des abréviations dans les nom de variable.